### ISSUE TITLE
Are you interested in ownership of our fork?

### ISSUE THREAD

**Issue by pixelass** [contentful/contentful-action#21]
We forked this action and fixed some issues we experienced while testing. Since we also wanted to add a few features we decided to do a full rewrite (still very much based on your original implementation) 

We, from [EVERNEST](http://evernest.com/), would like to offer you the ownership of this fork  and would be willing to prepare a PR including our changes. 
Our modified version is available on the [github marketplace](https://github.com/marketplace/actions/contentful-migration-automation)

You can read about the modifications [in the README](https://github.com/everdevs/contentful-action/blob/main/README.md)

tldr;

- optionally `set-alias`
- optionally `delete-feature` (delete sandbox-environment once merged)
- use arguments (`with`) instead of `env`
- allow environment naming patterns (i.e. `prod-[DD].[MM].[YY]-[hh][mm][ss]`)
- allows setting the content-type and field-id
- allows setting the migration-dir
- allows verbose logs (via `LOG_LEVEL="verbose"`)
- allows semantic versioning (i.e. `v1.2.3-rc.4`)

@Shy @stefanjudis

**Comment by Shy** [comment]
Hey @pixelass this is really great. Thanks for sharing this. I'll bring this up with Stefan in our next 1/1 and get back to you with what we decide.

**Comment by Shy** [comment]
Hey @pixelass. Would you be up for a call to chat. I want to make sure we're on the same page around this one. Could you shoot me an email to schedule something? I'm Shy@contentful.com

**Comment by Shy** [comment]
Just tagging that we've gone ahead and merged the pull request for this over in the Contentful userland organization. We'll be archiving this repo and pointing over to the new repo in a little bit. I'll be submitting the updated version to the GitHub marketplace, but the action is ready to go: [github.com/contentful-userland/contentful-action/](https://github.com/contentful-userland/contentful-action/)
